### ISSUE TITLE
feat(developer): Make it easier to create a new project and attach an existing KMN file to it feat: 

### ISSUE THREAD

**Issue by sewhite** [keymanapp/keyman#11847]
### Is your feature request related to a problem? Please describe.

I understand that in version 17 the recommended model is to have only one keyboard per project. To implement this in my existing projects, I need to create some new projects for some of my keyboards.  The process of doing this is cumbersome, for when you create a new project it assumes you want to create  a keyboard at the same time. Instead of creating a new keyboard, I should be able to select an existing keyboard and move it from the folder it was into the same folder as my new project.

I can do this by creating a new keyboard with the same name as my existing keyboard file, saving the new project and closing Keyman developer, then copying my existing keyboard into the new project folder, overwriting the empty keyboard file I just created. But this is a cumbersome process. 

### Describe the solution you'd like

When I create a new project, offer the choice of selecting an existing keyboard file to  import into this new project

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [X] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.326

### Operating system

Windows 11 22H2

### Device

Windows laptop

**Comment by sewhite** [comment]
Why not offer a fifth option to the four options when you click "new project". The fifth option would be "Import existing keyman keyboard from an older project". If you select it, it asks you to select the existing keyboard file, then offers the default location of a new folder with the existing keyboard name to create. 

For instance I have a folder in my Keyboard Developer projects called "Office archive" where I copied all the keyboard files I created on my office computer in the last five  years.  Say I want to do some work on a keyboard I called "Clavbur".  It would be great if I could click "New project" in Keyman  Developer, choose "Import existing keyboard",  browse to the clavbur.kmn file in my Office Archive, select it and have Developer name my new project "Clavbur", create the Clavbur folder inside the Projects folder, and copy "Clavbur.kmn" from my Office archive folder to the new project folder.
